### ISSUE TITLE
batch spam moderation may ban wrong users... unclear!!

### ISSUE THREAD

**Issue by jywarren** [jywarren/plots2#184]

**Comment by ebarry** [comment]
The "mass spam button" is still disabled -- is this intended?
![screen shot 2016-03-14 at 10 20 25 am](https://cloud.githubusercontent.com/assets/161439/13747395/6fff0d52-e9ce-11e5-8667-4066793c3e76.png)

**Comment by jywarren** [comment]
The issue is still not resolved, so yes!

**Comment by steviepubliclab** [comment]
There has been a lot of spam on the website lately, we've been thinking through how to simplify the moderation process and make it less of a burden for those who are helping out with moderating. I think making this button available to moderators would be a big help on this. What is blocking this from moving forward? 

adding @ebarry to follow

**Comment by jywarren** [comment]
Hi, nothing is blocking! Just nobody had gotten to it or prioritized it. I've unblocked the feature so it should work now, and wrote a new issue to add a test to protect the feature in the future: 

https://github.com/publiclab/plots2/issues/6569
